docs/diagrams/AddTagSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":Coach2K22Parser" as Coach2K22Parser LOGIC_COLOR
+participant ":AddTagCommandParser" as AddTagCommandParser LOGIC_COLOR
+participant "p:AddTagCommand" as AddTagCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("tag-add 1 t/friend")
+activate LogicManager
+
+LogicManager -> Coach2K22Parser : parseCommand("tag-add 1 t/friend")
+activate Coach2K22Parser
+
+create AddTagCommandParser
+Coach2K22Parser -> AddTagCommandParser
+activate AddTagCommandParser
+
+AddTagCommandParser --> Coach2K22Parser
+deactivate AddTagCommandParser
+
+Coach2K22Parser -> AddTagCommandParser : parse("1 t/friend")
```

Comment:
Might want to update the format here as well

Reply:
Fixed

Reply:
Fixed